src/SprykerEco/Shared/Payolution/PayolutionConfig.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Payolution;
+
+interface PayolutionConfig
```

Comment:
Should be class then just in case maybe. We soon have a concept here for code to also extend on project level.

Reply:
`class` is in `Zed`, here — ~stolen~ borrowed from `\Spryker\Shared\Queue\QueueConfig`

Reply:
Fair enough for now, but we drafted a plan of attack here for core to clean things up and provide more consistent recommendations for the near future.